src/renderer/components/player/SourceScraper.tsx

```diff
@@ -1529,12 +1509,12 @@ function loadHydrus(systemMessage: Function, config: Config, source: LibrarySour
           .catch((e) => {
             resolve(null);
           });
-      }
+      };
```

Comment:
@StarrHelixx There were a few semi-colons that seemed to be missing here and there in this part of the code? But if they shouldn't have been added, I'm happy to remove them!